demo/index.html

```diff
@@ -111,19 +110,14 @@
                 ctx.stroke();
             }
         };
-        let lastTimestamp = 0;
+
         demo.init(debugDraw);
-        (function loop() {
-            let fps = 60;
-            let now = Date.now();
-            let diff = now - lastTimestamp;
-            if (diff > (1000.0/fps)) {
-                ctx.clearRect(0, 0, 854, 400);
-                lastTimestamp = now;
-                demo.step();
-            }
-            window.requestAnimationFrame(loop);
-        })();
+
+        let fps = 60;
+        setInterval(function() {
```

Comment:
`setInteval` is not good idea for animation loop in browsers
https://stackoverflow.com/questions/38709923/why-is-requestanimationframe-better-than-setinterval-or-settimeout